workflows/prognostic_scream_run/tests/test_scream_config.py

```diff
@@ -0,0 +1,48 @@
+import os
+import scream_run
+import yaml
+import pytest
+from typing import List
+
+dirname = os.path.dirname(os.path.abspath(__file__))
+
+EXAMPLE_CONFIGS_DIR = os.path.join(dirname, "example_configs")
+
+
+@pytest.mark.parametrize(
+    "path, file", [(EXAMPLE_CONFIGS_DIR, "scream_ne30pg2.yaml")],
+)
```

Comment:
Is the parametrization really needed here since only one value is given for each of the parameters?  Is this just a personal style preference?  I notice that you do this in the later tests as well.

Reply:
I was expecting to have more then 1 test case, hence the choice of the parametrization. I will opt to keep it this way because we will very likely add more in the next PR that does the actual run.